### ISSUE TITLE
scraper items uniqueness not compared with any original csv file?

### ISSUE THREAD

**Issue by hiqsociety** [maximedrn/opensea-automatic-bulk-upload-and-sale#333]
scraper items uniqueness not compared with any original csv file?
how do i use scraper to check and compare missing from the original csv file?

well done by the way, thx for the scripts. helped a lot. this is my final area. i've uploaded 17k but missing 3k from total 20k but the scraper doesnt show which are missing but does show dup

**Comment by maximedrn** [comment]
Make sure you are using the latest version. When asked for the number of items the collection should contain, type 20000 in order to save the missing 3k.

**Comment by hiqsociety** [comment]
yes i did use latest but the 20000 is using what to justify and how does the 3k come about? 
1. i mean it doesnt say which original csv file to compare with, am i missing this step?
2. the 20000 does it follow numbering inside the items? coz my numbering is starting with 20000 like 
Nft #20000 to Nft #40000
the collection starts with 20000 because polygon max is 20000 items and i already have 1-20000 in another collection so splitting my collections into 2 collections coz of polygon limitations

need help with this last section. thx

**Comment by maximedrn** [comment]
It follows the numbering of your NFTs from lowest to highest.
Since it only scrapes 17000 items out of the 20000 that should be present (only 17k downloaded), it will create a missing file containing the missing 3000 from 37000 to 40000.

**Comment by hiqsociety** [comment]
i dont get missing files from some of the collections i have.
how do i get the missing? is there a way to match the csv file to the ones already scraped so i can get the missing ones?

**Comment by maximedrn** [comment]
What value do you type for the question `How many items should the collection contain?`?

**Comment by hiqsociety** [comment]
20000
the numbering is 20000 to 40000

**Comment by maximedrn** [comment]
Ok, the `missing()` function works from the lowest value to the value corresponding to the number of elements. So in your case, I think 40000 will create a file containing the missing.

**Comment by hiqsociety** [comment]
hi, thx.

now there's issue. it was working perfectly fine but now it's showing 
Something went wrong.
Expecting value line 1 columne 1 (char 0)

was working fine previously
i already did npm install, also copied to the opensea-collection-scraper folder

can u check again now? i think they upgraded their api
